trackintel/io/postgis.py

```diff
@@ -66,56 +94,52 @@ def write_positionfixes_postgis(
         How many entries should be written at the same time.
 
     if_exists : str, {'fail', 'replace', 'append'}, default 'fail'
-        What should happen if the table already exists.
+        How to behave if the table already exists.
+
+        - fail: Raise a ValueError.
+        - replace: Drop the table before inserting new values.
+        - append: Insert new values to the existing table.
 
     Examples
     --------
-    >>> df.as_positionfixes.to_postgis(conn_string, table_name)
+    >>> pfs.as_positionfixes.to_postgis(conn_string, table_name)
```

Comment:
would be nice to include also examples of calling this function directly, e.g., 
same for other write functions
```suggestion
    >>> pfs.as_positionfixes.to_postgis(conn_string, table_name)
    >>> ti.io.postgis.write_positionfixes_postgis(pfs, conn_string, table_name)
```